protocol/build.sbt

```diff
@@ -19,8 +19,8 @@
 // JSON DEPENDENCIES
 //
 libraryDependencies ++= Seq(
-  "com.typesafe.play" %% "play-json" % "2.3.10" excludeAll( // Apache v2
-      ExclusionRule(organization = "com.fasterxml.jackson.core")
+  "com.typesafe.play" %% "play-json" % "2.5.9" excludeAll( // Apache v2
+      //ExclusionRule(organization = "com.fasterxml.jackson.core")
```

Comment:
@lresende, in case the exclusion is to get a project working with toree client without explicitly depending on jackson, you could add the dependency without the exclusion to the "client" subproject's build.sbt